containers/Forms/CreateHabit.jsx

```diff
@@ -159,10 +159,8 @@ export default function CreateHabit(props) {
           </div>
         </form>
       </div>
-      <button
-        className={!visible ? styles.visible__form : styles.hidden__form}
-        onClick={(e) => toggleVisible()}
-      >
+      <button className={visible ? styles.hidden__button : styles.visible__button}
+              onClick={(e) => toggleVisible()}>
```

Comment:
where are u using 'e', if not remove

Reply:
that wasn't me , it was from main branch